packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewManager.kt

```diff
@@ -110,6 +110,11 @@ class ExpoImageViewManager(applicationContext: ReactApplicationContext) : Simple
     view.setTintColor(color)
   }
 
+  @ReactProp(name = "accessible")
+  fun setFocusable(view: ExpoImageView, accessible: Boolean) {
+    view.isFocusable = accessible // setFocusable(bool)
```

Comment:
I don't think we need that 🤔 
```suggestion
    view.isFocusable = accessible
```

Reply:
I believe you left this comment to inform that `view.isFocusable` Kotlin property covers Java `view.setFocusable()` setter? 🙂  I don't think it's necessary since IDE (Android Studio) should redirect you to proper definition when you Cmd+click on the property.

P.S. I know other props in this file directly use Java-style setters, but your approach is the right one :)

Reply:
There are method declarations:

[**1:**](https://developer.android.com/reference/android/view/View#isFocusable())
```java
public final boolean isFocusable ()
```
[**2:**](https://developer.android.com/reference/android/view/View#setFocusable(boolean))
```java
public void setFocusable (boolean focusable)
```
[**3:**](https://developer.android.com/reference/android/view/View#setFocusable(int))
```java
public void setFocusable (int focusable)
```

I left this comment, because it was unobvious for me, that `view.isFocusable` property refers to [**2nd**](https://developer.android.com/reference/android/view/View#setFocusable(boolean)) method.

Reply:
Ok, now I understand your intentions 😄 

Kotlin is smart and it can easily guess which function to call:
1. It noticed the assignment operator `=` so it must be setter. But there are two of them, which one?  🤔 
2. The `accessible` variable you want to assign is of type `Boolean`. Aha! That must be `setFocusable(boolean)` because it directly matches the type. 😅 

More on this [here](https://kotlinlang.org/docs/java-interop.html#getters-and-setters), but I have't found any docs covering setter overloading in Java-Kotlin interoperability.

P.S. I checked the Android Studio popup - it shows you the overload if you're not sure
![Screenshot 2021-08-25 at 13 43 17](https://user-images.githubusercontent.com/278340/130788691-ff0b71a0-93c6-42e8-a6e6-75d7aec89df5.png)

Reply:
>Kotlin is smart and it can easily guess which function to call:
> 1. It noticed the assignment operator = so it must be setter. But there are two of them, which one? 🤔
>~2. The accessible variable you want to assign is of type Boolean. Aha! That must be setFocusable(boolean) because it directly matches the type.~

>Edit: My mistake, it always expects setter to be of exact same type as getter, so if isFocusable() returns boolean, the assignment will expect setFocusable(boolean) even if there are multiple overloads.

>More on this here, but I have't found any docs covering setter overloading in Java-Kotlin interoperability.

I believe this proves my point, that it is not always obvious :smile: 

I see how method resolution generally works, however must admit that writing this line:
```kotlin
view.isFocusable = accessible
```
took me a minute :smile: (and a second from @lukmccall)

At first I was not aware, that two setters exist ([2](https://developer.android.com/reference/android/view/View#setFocusable(boolean)) & [3](https://developer.android.com/reference/android/view/View#setFocusable(int))), thus I attempted to write: 
```kotlin
view.focusable = accessible
```
and was quite baffled when Android Studio highlighted this as an error -- this is exactly the reason why this command was placed. However I won't insist on keeping it :smile: